Competition2018/src/main/java/competition/subsystems/elevator/commands/TalonCurrentMonitor.java

```diff
@@ -0,0 +1,41 @@
+package competition.subsystems.elevator.commands;
+
+import java.util.ArrayDeque;
+import xbot.common.controls.actuators.XCANTalon;
+
+public class TalonCurrentMonitor {
+
+    XCANTalon talon;
+    final int CURRENT_AVERAGING_WINDOW;
+    ArrayDeque<Double> currentHistory;
+
+    public TalonCurrentMonitor(XCANTalon talon) {
+        this.talon = talon;
+        currentHistory = new ArrayDeque<Double>();
+        CURRENT_AVERAGING_WINDOW = 25;
+    }
+
+    public double measureAverageCurrent() {
```

Comment:
Can we split this method into an `update`/`sample` method and a `calculateAverageCurrent`?